apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -329,12 +329,14 @@
             </tr>
           </thead>
           <tbody>
+            <%const fingerprint_colors = ['red2', 'gray3', 'blue3', 'orange3', 'brown1', 'green3']; %>
```

Comment:
How were these colors selected?

Reply:
These colors are the same given to events in the assessment event logs from `lib/assessment.sql`. I used the same general order/colors to maintain consistency on the page. Let me know if you would prefer to use different colors.

Reply:
I don't think there's a lot of value in trying to reuse the same colors since they're used for fundamentally different things between these two columns. I tested with this array and it seemed reasonable (and it's in rainbow order-ish):

```
['red2', 'orange2', 'green2', 'blue2', 'turquoise2', 'purple2']
```

Maybe this?

Reply:
These new colors look great! Updated to this color array in commit [0eb79a5](https://github.com/PrairieLearn/PrairieLearn/pull/9084/commits/0eb79a54350fff99bfd820989ce6c3ede01f14f2).

Reply:
Maybe green-orange-red? The first one is ok, the second is concerning, the third is a problem? :-D

I'm teasing, you don't need to take this seriously.